src/features/team-format-page.js

```diff
@@ -0,0 +1,132 @@
+var debug = require("../debug"),
+    vars = require("../vars"),
+    loadTeam = require("./team-load-team");
+
+module.exports = function () {
+
+    if(bttv.settings.get("formatTeamPage") !== true || $("body").attr("data-page") != "teams#show") return;
+    debug.log("Formatting team page");
+
+    var jquiCSS = $("<link>", {"href":"//ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/themes/dark-hive/jquery-ui.min.css", "type":"text/css", "rel":"stylesheet"}),
+        jquiJS  = $("<script>", {"src":"//ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/jquery-ui.min.js", "type":"text/javascript"}),
+        teamCSS = $("<link>", {"href":"//cdn.betterttv.net/style/stylesheets/betterttv-team-page.css?"+bttv.info.versionString(), "id":"betterTwitchTeams", "rel":"stylesheet", "type":"text/css"});
+    
+    if(bttv.settings.get("darkenedMode") === true) {
+        teamCSS = $("<link>", {"href":"//cdn.betterttv.net/style/stylesheets/betterttv-team-page-dark.css?"+bttv.info.versionString(), "id":"betterTwitchTeams", "rel":"stylesheet", "type":"text/css"});
+    }
+    $('body').append(jquiCSS, jquiJS, teamCSS);
+
+    $("h2").remove();
```

Comment:
h2 is for the "Members" heading text below the team Icon in the left hand column. It's literally the only place on the page it is used. But I do see your point, in case they use it else where in the future.

I'm a little busy now, but I will change it tonight/tomorrow to only remove that specific heading from it's parent container.